move-vm-backend/tests/move_vm.rs

```diff
@@ -502,3 +509,39 @@ fn dry_run_gas_strategy_doesnt_update_storage() {
         "received module although the dry run strategy was enabled"
     );
 }
+
+#[test]
+// TODO(rqnsom): this one should not panic once the native functions are implemented
+#[should_panic]
+// This test heavily depends on Move.toml files for the used Move packages.
+fn genesis_config_initializes_substrate_stdlib() {
+    let store = StorageMock::new();
+    let vm = Mvm::new(store /*, SimpleSubstrateApiMock {}*/).unwrap();
+    let gas = GasStrategy::Unmetered;
+
+    let stdlib = move_stdlib::substrate_stdlib_bundle();
+    let result = vm.publish_module_bundle(&stdlib, ADDR_STD, gas);
+    assert!(result.is_ok(), "failed to publish the substrate stdlib");
+}
+
+#[test]
+// This test heavily depends on Move.toml files for the used Move packages.
+fn genesis_config_initializes_stdlib() {
```

Comment:
We could have also failing test with publishing module without initialized stdlib.

Reply:
We have it already :smile:  - [publish_module_using_stdlib_full_fails](https://github.com/eigerco/substrate-move/blob/d215bc0820c5aa80b34161dc61e864ca9f935dc1/move-vm-backend/tests/move_vm.rs#L204)